back/engines/free/email_campaigns/spec/models/email_campaigns/campaigns/user_digest_spec.rb

```diff
@@ -65,23 +65,13 @@
     end
 
     it "only includes 'new' initiatives" do
-      proposed_status = create(:initiative_status_proposed)
+      create(:initiative_status_proposed)
 
       old_initiative = create(:initiative)
-      _old_initiative_status_change = create(
```

Comment:
I thought the tests would be a bit more natural if we don't create status changes right in tests.

Reply:
I agree. Much better :+1: